src/registrar/templates/home.html

```diff
@@ -17,9 +17,37 @@ <h2>
 <p>This is the .gov registrar.</p>
 
 {% if user.is_authenticated %}
-<p><a href="/openid/logout/">Click here to log out.</a></p>
+
+{% if domain_applications %}
+<h2>Your domain applications</h2>
+<table class="usa-table usa-table--borderless">
+  <caption class="sr-only">Your domain applications</caption>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Status</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for application in domain_applications %}
+      <tr>
+        <th>{{ application.requested_domain.website }}</th>
```

Comment:
**question (non-blocking)**: I invented the URL `/application/<id>` for the detail view of each domain application. Do we want to make this a link to that URL?

Reply:
I'm a bit uncertain about this change. I'm not opposed, but I can't find `/application/<id>` in `urls.py`, so I can't add a link to it yet. Otherwise `reverse` will complain `NoReverseMatch`.